### PR TITLE
Add Mismatches put request to api specs

### PR DESCRIPTION
**Pull request by itamargiv**
Bug: [T289133](https://phabricator.wikimedia.org/T289133)

### CODE REVIEW

File: docs/mismatchfinder-api-spec.yml [hunk 1]
```diff
@@ -38,6 +38,24 @@ paths:
         '422':
           $ref: '#/components/responses/ValidationErrors'
 
+    put:
```
Comment (Silvan-WMDE) [inline docs/mismatchfinder-api-spec.yml:41]: shouldn't this have the {id} parameter in the URL?

```
  /mismatches/{id}:
    put:
      tags: ['store']
      summary: Update Mismatch review status
      parameters:
        - in: path
          name: id
          schema:
            type: number
          required: true

```

**Review by Silvan-WMDE — CHANGES_REQUESTED** [review]
Thanks for spotting the missing spec!

File: docs/mismatchfinder-api-spec.yml (hunk 1 above)
Comment (itamargiv) [reply docs/mismatchfinder-api-spec.yml:41]: :facepalm: It for sure should...

**Review by Silvan-WMDE — APPROVED** [review]
nice, thanks!

**Pull request merged**